src/main/java/ch/wisv/events/app/controller/ICalController.java

```diff
@@ -33,11 +34,15 @@
 @RequestMapping(value = "/events")
 public class ICalController {
 
+    /**
+     * Field eventService
+     */
     private final EventService eventService;
 
     /**
      * Default constructor
      */
+    @Autowired
```

Comment:
[@Autowired is not needed when a Bean has only one constructor](https://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-spring-beans-and-dependency-injection.html)